packages/definitions-parser/test/git.test.ts

```diff
@@ -55,15 +55,20 @@ When removing packages, you should only delete files that are a part of removed
         { status: "D", file: "types/jest/index.d.ts" },
         { status: "D", file: "types/jest/jest-tests.d.ts" },
       ])
-    ).toEqual({ ok: jestNotNeeded });
+    ).toEqual({
+      errors: [
+        `Unexpected file added: oooooooooooops.txt
```

Comment:
this test had an error before and I incorrectly updated it when the error disappeared, also incorrectly.